examples/source_examples/smartsheets/sheets/connector.py

```diff
@@ -0,0 +1,72 @@
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests
+import json
+from datetime import datetime, timezone
+
+# Note: This code is currently configured to hit the getSheet endpoint and get data from 1 pre-defined sheet in Smartsheets that does not need pagination
+# getSheet endpoint: https://smartsheet.redoc.ly/tag/sheets?_gl=1*7tmndg*_gcl_au*MTYyNTczMDAxLjE3MzA0ODYyNzA.*_ga*MTAzMjIzODc4LjE3MzA0ODYyNzA.*_ga_ZYH7XNXMZK*MTczMDQ4NjI3MC4xLjEuMTczMDQ4Njg3Mi41My4wLjA.#operation/getSheet
```

Comment:
can we simplify it to `https://smartsheet.redoc.ly/tag/sheets#operation/getSheet`